runtime-network/aml/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=aml
+PKGSEC=libs
+PKGDES="Andri's Main Loop"
```

Comment:
What does that mean? Please be more descriptive.

Reply:
Fixed.